src/datasource_toolkit/forestadmin/datasource_toolkit/interfaces/actions.py

```diff
@@ -87,9 +89,19 @@ class ActionFieldType(enum.Enum):
     "UserDropdown",
 ]
 
+LayoutWidgetTypes = Literal["Separator", "Input"]
 
-class ActionField(TypedDict):
+
+class BaseActionFormElement(TypedDict):
     type: ActionFieldType
+
+
+class ActionLayoutElement(BaseActionFormElement):
+    component: LayoutWidgetTypes
+    fieldName: NotRequired[str]
```

Comment:
fieldName => fieldId
and I don't think it should be in ActionLayoutElement. It should belongs to ActionLayoutInputElement